consensus/service/src/test_utils/mod.rs

```diff
@@ -25,7 +26,16 @@ impl UntrustedInterfaces for TrivialTxManagerUntrustedInterfaces {
     }
 
     fn combine(&self, tx_contexts: &[&WellFormedTxContext], max_elements: usize) -> Vec<TxHash> {
-        tx_contexts
+        // Remove duplicates.
+        let uniques: HashSet<_> = tx_contexts.iter().cloned().collect();
+        let mut as_vec: Vec<_> = uniques.into_iter().collect();
+
+        // Sort, descending.
+        as_vec.sort();
```

Comment:
You could skip the HashSet and use https://doc.rust-lang.org/std/vec/struct.Vec.html#method.dedup

Reply:
Will do